core/lib/permissions.ts

```diff
@@ -18,3 +18,39 @@ export const getPubUsers = (permissions: PermissionPayload[]) => {
 	}
 	return users;
 };
+
+enum Privileges {
+	movePub,
+	createPub,
+	viewPub,
+	updatePubValues,
+	createRelatedPub,
+	createPubWithForm,
+	editPubWithForm,
+	createPubField,
+	archivePubField,
+	editPubField,
+	createPubType,
+	editPubType,
+	deletePubType,
+	createStage,
+	deleteStage,
+	addMoveConstraint,
+	removeMoveConstraint,
+	addActionToStage,
+	editAction,
+	removeActionFromStage,
+	addPubMember,
+	addStageMember,
+	addFormMember,
+	addCommunityMember,
+	createMemberGroup,
+	addMemberToGroup,
+	removeMemberFromGroup,
+	addCommunity,
+	createForm,
+	editForm,
+	archiveForm,
+	createApiToken,
+	revokeApiToken,
```

Comment:
One thing I would like to ask: could you make this enum use strings instead? Since we aren't doing any bit masking or anything, I don't see much value to leaving the enum members as integers, and have found that often kind of confusing.

That way `Privileges.movePub === "movePub"`, rather than `Privileges.movePub === 0`

This would entail this.
```suggestion
    movePub = "movePub",
    createPub = "createPub",
    viewPub = "viewPub",
    updatePubValues = "updatePubValues",
    createRelatedPub = "createRelatedPub",
    createPubWithForm = "createPubWithForm",
    editPubWithForm = "editPubWithForm",
    createPubField = "createPubField",
    archivePubField = "archivePubField",
    editPubField = "editPubField",
    createPubType = "createPubType",
    editPubType = "editPubType",
    deletePubType = "deletePubType",
    createStage = "createStage",
    deleteStage = "deleteStage",
    addMoveConstraint = "addMoveConstraint",
    removeMoveConstraint = "removeMoveConstraint",
    addActionToStage = "addActionToStage",
    editAction = "editAction",
    removeActionFromStage = "removeActionFromStage",
    addPubMember = "addPubMember",
    addStageMember = "addStageMember",
    addFormMember = "addFormMember",
    addCommunityMember = "addCommunityMember",
    createMemberGroup = "createMemberGroup",
    addMemberToGroup = "addMemberToGroup",
    removeMemberFromGroup = "removeMemberFromGroup",
    addCommunity = "addCommunity",
    createForm = "createForm",
    editForm = "editForm",
    archiveForm = "archiveForm",
    createApiToken = "createApiToken",
    revokeApiToken = "revokeApiToken",
```

Reply:
good point!